README.md

```diff
@@ -11,22 +11,24 @@
 
 # DSP-TOOLS - DaSCH Service Platform Tools
 
-DSP-TOOLS is a command line tool that helps you to interact with the DaSCH Service Platform API. 
+DSP-TOOLS is a command line tool that helps you to interact with the DaSCH Service Platform (DSP) API. 
 This document is intended for developers who want to work with the code of DSP-TOOLS. 
 
 | <center>Hint</center>                                                                                                                                                                |
 |:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| This technical document was written as a guide for developers. For the end user documentation, please consult [https://docs.dasch.swiss](https://docs.dasch.swiss/latest/DSP-TOOLS). |
+| This technical document was written as a guide for developers. For the end user documentation, 
+please consult [https://docs.dasch.swiss](https://docs.dasch.swiss/latest/DSP-TOOLS). |
```

Comment:
This additional line break breaks the markdown table. Or was it your intention that from "please" on, it is in a new row?

If you want it like this, then I suggest writing the entire 2nd sentence in the new row.

Reply:
No i had trouble with the line length but i did not intend for that break to be displayed.